python/py-rawpy/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                py-rawpy
+version             0.17.2
+revision            0
+license             MIT
+
+github.setup        letmaik rawpy v0.17.2
+fetch.type          git
```

Comment:
why do you need this?

Reply:
You mean downloading from github ? There is no source on Pypi, only pre-built wheels.